### ISSUE TITLE
Using template Nativescript-Vue-Web error in iOS

### ISSUE THREAD

**Issue by JoaquinGL** [nativescript-vue/vue-cli-plugin-nativescript-vue#9]
If you use this template, in the `App.vue` file, needs to import the component HelloWorld. Otherwise there will be an error when you try the app in iOS.

Needs proper import 
```
import HelloWorld from './components/HelloWorld';
components: {
   HelloWorld,
}
```

**Comment by jawa-the-hutt** [comment]
@JoaquinGL Can you give me some detail on what options you used to create the Vue project?  For example:  Typescript, CSS Preprocessors, etc?

Also, I need the same for the plugin:  new app, native only or dual, etc.

Will help me try and reproduce exactly what you were using.

**Comment by JoaquinGL** [comment]
Sure, for the vue project I just create a new porject:
`vue create my-app`

Then for the plugin I used everything by default and the template was `Nativescript-Vue-Web`

**Comment by jawa-the-hutt** [comment]
This should now be fixed in 0.0.15 which was just pushed to the npm registry.
